bundles/org.openhab.io.homekit/pom.xml

```
@@ -23,7 +23,7 @@
     <dependency>
       <groupId>io.github.hap-java</groupId>
       <artifactId>hap</artifactId>
-      <version>1.2.0.OH</version>
+      <version>2.0.0-snapshot</version>
```

```suggestion
      <version>2.0.0.OH</version>
```

I have released hap-java 2.0.0.OH which includes your changes and some minor tooling fixes. Please check if that works. My local build succeeds with that one. Once hap-java 2.0.0 is released, we can switch to that.

My goal would be to have these nice improvements in openHAB 2.5.5, which is possible since you implemented full backward-compatibility. Thanks.

---

👍 looks good. thanks for doing the special hap java release!